src/js/types/Package.ts

```diff
@@ -0,0 +1,13 @@
+import { PackageVersion } from "#SRC/js/types/PackageVersion";
```

Comment:
why not having them in `data/cosmos`?

Reply:
That would work for me. I was following the example in the jobs plugin and we did the same thing for SDKPlans in the services plugin.

Reply:
That's true and both jobs and services are separate plugins with a separate scope. Here we are in dcos-ui context, not a plugin. I wonder if we should be more specific when not in a plugin context.

Reply:
Roger, moved.